yapftests/yapf_test.py

```diff
@@ -15,6 +15,7 @@
 """Tests for yapf.yapf."""
 
 import io
+from re import match
```

Comment:
How about "This is my project and I want it that way, and it keeps the namespace cleaner"?